src/app/errors/routes.ts

```diff
@@ -3,13 +3,14 @@ import { Route } from '../navigation/types';
 
 const CATCH_ALL = '/(.*)';
 
-export const notFound: Route<undefined> = {
+export const notFound: Route<undefined> & { getFullUrl: () => string } = {
```

Comment:
I'm not sure if there is a benefit of altering this, can't the link be outside of it?

Reply:
Maybe we could just create left only `getUrl` and modify it to contain full url?